spec/dummy/app/controllers/api_controller.rb

```diff
@@ -4,4 +4,9 @@ def data
     render json: ["some", "server", "data"]
   end
 
+  def single_endpoint
+    user = "user number #{params[:number]}"
+    render json: [user]
+  end
```

Comment:
@pascalwengerter you never use this response, right? probably you want to remove the logic, as it is confusing, what it intends to do

Reply:
I think I used it to test locally, but since I haven't included it in the `spec/usage/components/link_spec.rb` it can be safely removed